docs/UsersGuide/source/CustomizingTheWorkflow/LAMGrids.rst

```diff
@@ -160,45 +162,260 @@ The following is an example of a code stanza for "NEW_GRID" to be added to ``pre
    
    #  The method used to generate the grid. This example is specifically for the "ESGgrid" method.
 
-      GRID_GEN_METHOD: "ESGgrid"
+
+     GRID_GEN_METHOD: "ESGgrid"
    
    #  ESGgrid parameters:
 
-      ESGgrid_LON_CTR: -97.5
-      ESGgrid_LAT_CTR: 38.5
-      ESGgrid_DELX: 25000.0
-      ESGgrid_DELY: 25000.0
-      ESGgrid_NX: 200
-      ESGgrid_NY: 112
-      ESGgrid_PAZI: 0.0
-      ESGgrid_WIDE_HALO_WIDTH: 6
+     ESGgrid_LON_CTR: -97.5
+     ESGgrid_LAT_CTR: 38.5
+     ESGgrid_DELX: 25000.0
+     ESGgrid_DELY: 25000.0
+     ESGgrid_NX: 200
+     ESGgrid_NY: 112
+     ESGgrid_PAZI: 0.0
+     ESGgrid_WIDE_HALO_WIDTH: 6
 
    #  Forecast configuration parameters:
 
-      DT_ATMOS: 40
-      LAYOUT_X: 5
-      LAYOUT_Y: 2
-      BLOCKSIZE: 40
+     DT_ATMOS: 40
+     LAYOUT_X: 5
+     LAYOUT_Y: 2
+     BLOCKSIZE: 40
 
    #  Parameters for the write-component (aka "quilting") grid. 
 
-      QUILTING:
-         WRTCMP_write_groups: 1
-         WRTCMP_write_tasks_per_group: 2
-         WRTCMP_output_grid: "lambert_conformal"
-         WRTCMP_cen_lon: -97.5
-         WRTCMP_cen_lat: 38.5
-         WRTCMP_lon_lwr_left: -121.12455072
-         WRTCMP_lat_lwr_left: 23.89394570
+     QUILTING:
+       WRTCMP_write_groups: 1
+       WRTCMP_write_tasks_per_group: 2
+       WRTCMP_output_grid: "lambert_conformal"
+       WRTCMP_cen_lon: -97.5
+       WRTCMP_cen_lat: 38.5
+       WRTCMP_lon_lwr_left: -121.12455072
+       WRTCMP_lat_lwr_left: 23.89394570
 
    #  Parameters required for the Lambert conformal grid mapping.
 
-         WRTCMP_stdlat1: 38.5
-         WRTCMP_stdlat2: 38.5
-         WRTCMP_nx: 197
-         WRTCMP_ny: 107
-         WRTCMP_dx: 25000.0
-         WRTCMP_dy: 25000.0
+       WRTCMP_stdlat1: 38.5
+       WRTCMP_stdlat2: 38.5
+       WRTCMP_nx: 197
+       WRTCMP_ny: 107
+       WRTCMP_dx: 25000.0
+       WRTCMP_dy: 25000.0
 
 .. note:: 
    The process above explains how to create a new *predefined* grid, which can be used more than once. If a user prefers to create a custom grid for one-time use, the variables above can instead be specified in ``config.yaml``, and ``PREDEF_GRID_NAME`` can be set to a null string. In this case, it is not necessary to modify ``valid_param_vals.yaml`` or ``predef_grid_params.yaml``. Users can view an example configuration file for a custom grid `here <https://github.com/ufs-community/ufs-srweather-app/blob/develop/tests/WE2E/test_configs/wflow_features/config.custom_ESGgrid.yaml>`__.
+
+.. _VerticalLevels:
+
+Changing the Number of Vertical Levels
+========================================
+
+The four supported predefined grids included with the SRW App have 65 vertical levels. However, advanced users may wish to vary the number of vertical levels in the grids they are using, whether these be the predefined grids or a user-generated grid. Varying the number of vertical levels requires
```

Comment:
I would say that the four grids "are configured to run with 65 levels by default".